src/OutputColorizer.Tests/OutputColorizer.Tests.csproj

```diff
@@ -16,21 +16,22 @@
     <ReferencePath>$(ProgramFiles)\Common Files\microsoft shared\VSTT\$(VisualStudioVersion)\UITestExtensionPackages</ReferencePath>
     <IsCodedUITest>False</IsCodedUITest>
     <TestProjectType>UnitTest</TestProjectType>
+    <BaseIntermediateOutputPath>..\..\bin\obj\net45.Tests\</BaseIntermediateOutputPath>
```

Comment:
We don't, but if you don't add this, then the bin and obj dirs will be located under src/ folder which is usually not ideal. If we follow what we do on the product, we usually try to have the src tree always clean of all output files, so if we want to clean all the build outputs is as easy as simple deleting the root bin folder. Again, we don't need it so I'm fine if you want to move it back on how it was.